### PR TITLE
Feature/icons

### PR DESCRIPTION
**Pull request by NuriyaAkh**
## Describe your changes
Create components for every Icon that is used in figma
## Issue ticket number and link
closes 61
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by NuriyaAkh** [comment]
Settings icon is not rendering correctly

File: src/components/icons/add.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const Add = () => {
+  return (
+    <svg
+      width="14"
+      height="14"
```
Comment (jahorwitz) [inline src/components/icons/add.jsx:5]:
```suggestion
const Add = ({className, width = "16", height = "16" }) => {
  return (
    <svg
      width={width}
      height={height}
      className={className}
```

**Review by jahorwitz — CHANGES_REQUESTED** [review]
Just one change, but it'll be in several places:

Could you please implement a standard for all of these icons that each of them accepts props for `className`, `width` (default to `"16"`), `height` (also default to `"16"`), and pass those props to the `<svg` element? This will make these icons just a bit more flexible. For any of the ones we are currently using, make sure to pass in values to override the defaults if needed. I made a comment for `add.jsx` to show an example of what I mean.

**Comment by kamal-ganiev** [comment]
Leaving a comment for changes made on last commit:
1. Add "active" attribute to Home, Profile and Notifications
2. Add conditional statement for fill-opacity.
3. Change names of components to avoid match naming with another components ( ex. Home to HomeIcon, <Home /> already exists as layout ).
4. Remove nav-tray-icons component, because we don't need it anymore.

**Review by kamal-ganiev — APPROVED** [review]
Thank you for changing names back, the only thing, we might need to remove nav-link-icon, since we get Profile icon from corresponding file.

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**